components/src/dynamo/vllm/handlers.py

```
@@ -825,6 +825,382 @@ async def stop_profile(self, body: dict) -> dict:
+                    )
+            self._paused = True
+            logger.info(
+                f"[RL] Engine paused (generation quiesced, mode={mode}, clear_cache={clear_cache})"
+            )
+            return {
+                "status": "ok",
+                "message": "Engine paused",
+                "mode": mode,
+                "clear_cache": clear_cache,
+            }
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)